src/main/java/io/confluent/idesidecar/restapi/application/SidecarInfo.java

```diff
@@ -133,6 +134,8 @@ public Optional<VsCode> vsCode() {
     return vscode;
   }
 
+  public String osArch() { return osArch; }
```

Comment:
Ah, sorry, I just noticed that this is a one-liner. Let's please split it up like so:
```suggestion
  public String osArch() {
    return osArch;
  }
```